### PR TITLE
information-schema: add client errors docs

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
This adds docs for https://github.com/pingcap/tidb/pull/22382

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/5062
- Other reference link(s): https://github.com/pingcap/docs/pull/5251

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/client-errors-summary-by-user.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+title: CLIENT_ERRORS_SUMMARY_BY_USER
+summary: 了解 information_schema 表 `CLIENT_ERRORS_SUMMARY_BY_USER`。
+---
+
+# CLIENT_ERRORS_SUMMARY_BY_USER
```
Comment (Joyinqin) [inline information-schema/client-errors-summary-by-user.md:6]: 英文此处需要修改

File: information-schema/client-errors-summary-global.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+title: CLIENT_ERRORS_SUMMARY_GLOBAL
+summary: 了解 information_schema 表 `CLIENT_ERRORS_SUMMARY_GLOBAL`。
+---
+
+# CLIENT_ERROR_SUMMARY_GLOBAL
```
Comment (Joyinqin) [inline information-schema/client-errors-summary-global.md:6]: 英文此处需要修改

File: information-schema/client-errors-summary-by-user.md (hunk 1 above)
Comment (TomShawn) [reply information-schema/client-errors-summary-by-user.md:6]: 已修改：https://github.com/pingcap/docs/pull/5251

File: information-schema/client-errors-summary-global.md (hunk 2 above)
Comment (TomShawn) [reply information-schema/client-errors-summary-global.md:6]: 已修改：pingcap/docs#5251

File: information-schema/client-errors-summary-by-host.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+`CLIENT_ERRORS_SUMMARY_BY_HOST` 表提供了连接到 TiDB 服务器的客户端所返回的 SQL 错误和警告汇总。这些错误和警告包括：
+
+* 格式错误的 SQL 语句。
+* 除以零错误。
+* 尝试插入超出范围或重复的键值。
+* 权限错误。
+* 表不存在。
+
+MySQL 服务器协议会返回以上错误到客户端，此时应用程序应在客户端采取适当操作。information_schema 表 `CLIENT_ERRORS_SUMMARY_BY_HOST` 提供了一种有效方法，能够在应用程序没有正确处理（或记录）TiDB 服务器返回的错误的情况下检查错误。
```
Comment (djshow832) [inline information-schema/client-errors-summary-by-host.md:16]:
```suggestion
MySQL 服务器协议会返回以上错误到客户端，此时应用程序应在客户端采取适当操作。`information_schema`.`CLIENT_ERRORS_SUMMARY_BY_HOST` 表提供了一种有效方法，能够在应用程序没有正确处理（或记录）TiDB 服务器返回的错误的情况下检查错误。
```

File: information-schema/client-errors-summary-by-host.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+| LAST_SEEN     | timestamp     | YES  |      | NULL    |       |
++---------------+---------------+------+------+---------+-------+
+7 rows in set (0.00 sec)
+```
+
+字段说明如下：
+
+* `HOST`：客户端的远程主机。
+* `ERROR_NUMBER`：返回的与 MySQL 兼容的错误号。
+* `ERROR_MESSAGE`：与错误号匹配的错误消息（预处理语句形式）。
```
Comment (djshow832) [inline information-schema/client-errors-summary-by-host.md:53]:
```suggestion
* `ERROR_NUMBER`：返回的与 MySQL 兼容的错误码。
* `ERROR_MESSAGE`：与错误码匹配的错误消息（预处理语句形式）。
```

File: information-schema/client-errors-summary-by-user.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+`CLIENT_ERRORS_SUMMARY_BY_USER` 表提供了连接到 TiDB 服务器的客户端所返回的 SQL 错误和警告的汇总。这些错误和警告包括：
+
+* 格式错误的 SQL 语句。
+* 除以零错误。
+* 尝试插入超出范围或重复的键值。
+* 权限错误。
+* 表不存在。
+
+MySQL 服务器协议会返回以上错误到客户端，此时应用程序应在客户端采取适当操作。information_schema 表 `CLIENT_ERRORS_SUMMARY_BY_USER` 提供了一种有效方法，能够在应用程序没有正确处理（或记录）TiDB 服务器返回的错误的情况下检查错误。
```
Comment (djshow832) [inline information-schema/client-errors-summary-by-user.md:16]:
```suggestion
MySQL 服务器协议会返回以上错误到客户端，此时应用程序应在客户端采取适当操作。`information_schema`.`CLIENT_ERRORS_SUMMARY_BY_USER` 表提供了一种有效方法，能够在应用程序没有正确处理（或记录）TiDB 服务器返回的错误的情况下检查错误。
```

File: information-schema/client-errors-summary-by-user.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+| LAST_SEEN     | timestamp     | YES  |      | NULL    |       |
++---------------+---------------+------+------+---------+-------+
+7 rows in set (0.00 sec)
+```
+
+字段说明如下：
+
+* `USER`：已认证的用户。
+* `ERROR_NUMBER`：返回的与 MySQL 兼容的错误号。
+* `ERROR_MESSAGE`：与错误号匹配的错误消息（预处理语句形式）。
```
Comment (djshow832) [inline information-schema/client-errors-summary-by-user.md:52]:
```suggestion
* `ERROR_NUMBER`：返回的与 MySQL 兼容的错误码。
* `ERROR_MESSAGE`：与错误码匹配的错误消息（预处理语句形式）。
```

File: information-schema/client-errors-summary-global.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+`CLIENT_ERROR_SUMMARY_GLOBAL` 表提供了连接到 TiDB 服务器的客户端所返回的 SQL 错误和警告汇总。这些错误和警告包括：
+
+* 格式错误的 SQL 语句。
+* 除以零错误。
+* 尝试插入超出范围或重复的键值。
+* 权限错误。
+* 表不存在。
+
+MySQL 服务器协议会返回以上错误到客户端，此时应用程序应在客户端采取适当操作。information_schema 表 `CLIENT_ERRORS_SUMMARY_GLOBAL` 提供了一种高级概述，在应用程序无法正确处理（或记录）TiDB 服务器返回的错误的情况下很有用。
```
Comment (djshow832) [inline information-schema/client-errors-summary-global.md:16]: Ditto

File: information-schema/client-errors-summary-global.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+| FIRST_SEEN    | timestamp     | YES  |      | NULL    |       |
+| LAST_SEEN     | timestamp     | YES  |      | NULL    |       |
++---------------+---------------+------+------+---------+-------+
+6 rows in set (0.00 sec)
+```
+
+字段说明如下：
+
+* `ERROR_NUMBER`：返回的与 MySQL 兼容的错误编号。
+* `ERROR_MESSAGE`：与错误编号相匹配的错误信息（以预准备语句形式）。
```
Comment (djshow832) [inline information-schema/client-errors-summary-global.md:44]: Ditto

File: information-schema/client-errors-summary-by-host.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
+---
+title: CLIENT_ERRORS_SUMMARY_BY_HOST
+summary: 了解 information_schema 表 `CLIENT_ERRORS_SUMMARY_BY_HOST`。
+---
+
+# CLIENT_ERROR_SUMMARY_BY_HOST
+
+`CLIENT_ERRORS_SUMMARY_BY_HOST` 表提供了连接到 TiDB 服务器的客户端所返回的 SQL 错误和警告汇总。这些错误和警告包括：
```
Comment (lilin90) [inline information-schema/client-errors-summary-by-host.md:8]: According to the EN version, it seems that it's the following meaning.
```suggestion
`CLIENT_ERRORS_SUMMARY_BY_HOST` 表汇总了已返回给连接到 TiDB 服务器的客户端的 SQL 错误和警告。这些错误和警告包括：
```

File: information-schema/client-errors-summary-by-host.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+
+* 格式错误的 SQL 语句。
+* 除以零错误。
+* 尝试插入超出范围或重复的键值。
+* 权限错误。
+* 表不存在。
+
+MySQL 服务器协议会返回以上错误到客户端，此时应用程序应在客户端采取适当操作。`information_schema`.`CLIENT_ERRORS_SUMMARY_BY_HOST` 表提供了一种有效方法，能够在应用程序没有正确处理（或记录）TiDB 服务器返回的错误的情况下检查错误。
```
Comment (lilin90) [inline information-schema/client-errors-summary-by-host.md:16]:
```suggestion
以上错误通过 MySQL 服务器协议返回给客户端，此时应用程序应在客户端采取适当操作。`information_schema`.`CLIENT_ERRORS_SUMMARY_BY_HOST` 表提供了一种有效方法，能够在应用程序没有正确处理（或记录）TiDB 服务器返回的错误的情况下检查错误。
```

File: information-schema/client-errors-summary-by-host.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+
+由于 `CLIENT_ERRORS_SUMMARY_BY_HOST` 会基于每个远程主机汇总错误，在诊断其中一台应用程序服务器比其他服务器生成更多错误的情况时很有用。可能的情况包括：
+
+* 过时的 MySQL 客户端库。
+* 过时的应用程序（可能是在推出新部署时遗漏了此服务器）。
+* 用户权限中“主机”部分的使用不正确。
+* 网络连接不可靠，导致更多超时或断开的连接。
```
Comment (lilin90) [inline information-schema/client-errors-summary-by-host.md:23]:
```suggestion
* 网络连接不可靠，导致更多超时或连接断开。
```

File: information-schema/client-errors-summary-by-host.md [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+
+可以使用 `FLUSH CLIENT_ERRORS_SUMMARY` 语句重置汇总的计数。汇总是每个 TiDB 服务器的本地数据，并且只保留在内存中。如果 TiDB 服务器重新启动，会丢失汇总。
```
Comment (lilin90) [inline information-schema/client-errors-summary-by-host.md:25]:
```suggestion
可以使用 `FLUSH CLIENT_ERRORS_SUMMARY` 语句重置汇总的计数。所汇总的是每个 TiDB 服务器的本地数据，并且只保留在内存中。如果 TiDB 服务器重新启动，会丢失汇总信息。
```

**Review by lilin90 — COMMENTED** [review]
I only added comments to `information-schema/client-errors-summary-by-host.md`, but since the other two files have almost the same description, please also update `client-errors-summary-by-user.md` and `client-errors-summary-global.md`.

**Comment by Joyinqin** [comment]
@lilin90, @TomShawn PTAL again, thx~

**Review by yikeke — APPROVED** [review]

**Pull request merged**